src/app/components/Image/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const getText = block => block.model.blocks[0].model.blocks[0].model.text;
+
+const Image = ({ blocks }) => {
+  const subBlocks = blocks[0].model.blocks;
+  const rawImageBlock = subBlocks.filter(block => block.type === 'rawImage')[0];
+  const altTextBlock = subBlocks.filter(block => block.type === 'altText')[0];
+  const captionBlock = subBlocks.filter(block => block.type === 'caption')[0];
+
+  if (!rawImageBlock) {
+    return null;
+  }
+
+  const rawImageLocator = rawImageBlock.model.locator;
+  const altText = altTextBlock ? getText(altTextBlock) : '';
+  const caption = captionBlock ? getText(captionBlock) : null;
+  const rawImageSrc = `https://ichef.bbci.co.uk/news/640${rawImageLocator}`;
+
+  return (
+    <figure>
+      <img alt={altText} src={rawImageSrc} />
+      {caption ? <figcaption>{caption}</figcaption> : null}
+    </figure>
+  );
+};
+
+Image.propTypes = {
```

Comment:
Ignore this code duplication, we should do it in another issue. We should have a single abstraction for props that are article data and the work would make this PR vastly more convoluted.

Reply:
There's an issue here to capture this work https://github.com/bbc/simorgh/issues/141